Sa additional ipv6 (#7875)
ovh/docs: pages/bare_metal_cloud/dedicated_servers/configure-an-ipv6-in-a-vrack/guide.fr-fr.md

**Ce guide se concentre sur la configuration des blocs d’adresses Additional IPv6 au sein d’un réseau vRack.**

> [!primary]
>

> > Le vRack prend en charge le routage public IPv4 et IPv6 avec des blocs d’adresses Additional IP. Vous pouvez trouver des instructions vous indiquant comment configurer des blocs IPv4 dans ce guide: [Configurer un bloc IP dans le vRack](/pages/bare_metal_cloud/dedicated_servers/configuring-an-ip-block-in-a-vrack/).

Reviewer: ```suggestion
> Le vRack prend en charge le routage public IPv4 et IPv6 avec des blocs d’adresses Additional IP. Retrouvez les instructions sur la configuration de blocs IPv4 dans ce guide: [Configurer un bloc IP dans le vRack](/pages/bare_metal_cloud/dedicated_servers/configuring-an-ip-block-in-a-vrack/).
```

>

> [!primary]
>

> > Cet article parle d'Additional IP. Si vous avez besoin d'assistance sur la configuration de Primary IP, nous vous suggérons les articles suivants:<br> 

Reviewer: ```suggestion
> Cet article parle d'Additional IP. Si vous avez besoin d'assistance sur la configuration d'une adresse IP principale, nous vous suggérons les articles suivants :<br> 
```

Author: Made requested edit.

> > Cet article parle de la configuration d'Additional IP sur un réseau vRack. Si vous cherchez des conseils sur la configuration d'Additional IP avec une adresse IP principale (sur l'interface réseau publique), nous vous suggérons de consulter les articles suivants :<br> 

Reviewer: ```suggestion
> Cet article détaille la configuration d'adresses Additional IP sur un réseau vRack. Si vous cherchez des instructions sur la configuration d'adresses Additional IP avec une adresse IP principale (sur l'interface réseau publique), consultez les articles suivants :<br> 
```

L'exemple ci-dessus montre deux hôtes avec leurs interfaces côté vRack configurées avec des adresses publiques IPv6. Un hôte possède une configuration manuelle, tandis qu’un autre dispose d'une adresse IP attribuée automatiquement en SLAAC. Toutes les adresses IP appartiennent au premier sous-réseau /64 d'un bloc /56 d'Additional IPv6 publiques donné. Les deux utilisent l'interface vRack pour la connectivité IPv6 publique.

> La passerelle par défaut pour le premier sous-réseau /64 (celui qui est bridgé) est la première adresse du bloc /56. Dans cet exemple, la passerelle est `2001:41d0:abcd:ef00::1`. Celle-ci est distribuée via SLAAC, mais doit être configurée manuellement (en tant que route par défaut) si le SLAAC est désactivé - voir **Configuration IP statique** ci-dessous.

Reviewer: ```suggestion
La passerelle par défaut pour le premier sous-réseau /64 (celui qui est bridgé) est la première adresse du bloc /56. Dans cet exemple, la passerelle est `2001:41d0:abcd:ef00::1`. Celle-ci est distribuée via SLAAC, mais doit être configurée manuellement (en tant que route par défaut) si le SLAAC est désactivé (voir la section **Configuration IP statique** ci-dessous).
```
